packages/roosterjs-editor-dom/lib/snapshots/clearSnapshotsAfterCurrent.ts

```diff
@@ -0,0 +1,17 @@
+import canMoveCurrentSnapshot from './canMoveCurrentSnapshot';
+import { Snapshots } from 'roosterjs-editor-types';
+
+/**
+ * Clear all snapshots after the current one
+ * @param snapshots The snapshots data structure to clear
+ */
+export default function clearSnapshotsAfterCurrent(snapshots: Snapshots) {
```

Comment:
nit: the name `clearSnapshotsAfterCurrent` is a bit confusing to me, suggest change to something like `clearProceedingSnapshots`?

Reply:
sounds good.